test/utils.js

```diff
@@ -0,0 +1,15 @@
+import * as bowser from 'bowser'
```

Comment:
Looking at both projects, bowser looks more mature, well tested, and actively maintained. I would change to bowser.